src/faq.html

```diff
@@ -6,7 +6,7 @@
   - name: General
     faqs:
       - title: >
-          Can I use the EDATec Raspberry Pi Compute Module 5 Active Cooler?
+          Can I use the Raspberry Pi Compute Module 5 Active Cooler?
```

Comment:
Afaik, the active cooler for CM5 is not an official one but from that third party EDATec, so probably make sense to leave this here?

It does seem like an unrelated change anyways? :thinking: